src/ConfigureMentions.php

```diff
@@ -42,15 +43,23 @@ private function configureUserMentions(Configurator $config)
         $tagName = 'USERMENTION';
 
         $tag = $config->tags->add($tagName);
-        $tag->attributes->add('username');
         $tag->attributes->add('displayname');
+        $tag->attributes->add('slug');
```

Comment:
forgot about this, totally unnecessary to save the slug, we can set it when updating the xml before formatting.

Reply:
Great! The less we store permanently, the better :grin: